src/main/webapp/app/course/competencies/competency-accordion/competency-accordion.component.scss

```diff
@@ -0,0 +1,94 @@
+.competency-accordion-header {
+    background-color: var(--overview-light-background-color);
+    color: var(--bs-body-color);
+    cursor: pointer;
+    padding: 5px;
+    width: 100%;
+    border: none;
+    text-align: left;
+    outline: none;
+    font-size: 15px;
```

Comment:
in general we should stop defining custom font-sizes and use bootstrap classes that we use uniform font sizes throughout Artemis

Reply:
Removed since it was anyways unused